src/fprime/cookiecutter_templates/cookiecutter-fprime-component/{{cookiecutter.component_name}}/docs/sdd.md

```diff
@@ -31,9 +29,25 @@ Add component states in the chart below
 Add sequence diagrams here
 
 ## Parameters
+| Name | Description |
```

Comment:
Should these be driven by the choices made?

Reply:
At some point I want to auto-generate these sections of the SDD....so maybe it doesn't matter.

Reply:
We can add that really easily, my thought process on not doing it is that the presence of the information "there is nothing" has a stronger meaning than the lack of information. So the presence of an empty _Parameters_ table is more information than the lack of a _Parameter_ table.

Reply:
Agreed.